### ISSUE TITLE
Tageler detail view, only shows one Einheit

### ISSUE THREAD

**Issue by peschmae** [tageler/tageler-frontend#46]
Even when multiple units are chosen when creating a tageler, only the first one is shown in the detail view

**Comment by peschmae** [comment]
I propose to just list all units, comma separated, and not every unit on it's own line.

To achieve that, we can just change
`<p id="group" class="card-text" [innerHTML]="tageler.group[0].toString().split(',').join('<br \>')"></p>`

to
`<p id="group" class="card-text" [innerHTML]="tageler.group[0].toString()"></p>`

**Comment by peschmae** [comment]
I seem to have changed the details view, in the user context, instead of the admin context...

Should have probably dug a bit deeper into the source before commiting...

**Comment by peschmae** [comment]
This is related to #47, since übungsfrei tageler for multiple units are somehow saved differently than normal tagelers for multiple groups.

And it also happens in the user context, but only for übungsfrei, for multiple units, and can be remedied by editing the tageler, and saving it again

**Comment by peschmae** [comment]
Looking at the network traffic when saving a normal tageler, and a tageler that is übungsfrei, the payload looks identical.

Within the API, I can't see anything handling the groups on the server side, beside assigning it to the model, before saving it into mongodb.

This will be a hard one to debug...

**Comment by peschmae** [comment]
There is a difference, when creating a multi-unit Übungsfrei, and a normal multi-unit Tageler, in the data transmited to the API, so the issue lies within the frontend code.

Normal Tageler
```
{
  "title": "Multi-Unit 2",
  "text": "Shouldn't change much",
  "group": [
    [
      "Obsidian",
      "Turmalin",
      "Gürbe",
      "Bratwurscht",
      "Baghira",
      "Rikki-Tikki"
    ]
  ],
  "start": "2018-01-26T11:00:00.000Z",
  "end": "2018-01-26T14:00:00.000Z",
  "bringAlong": "BPMSTZ und Zvieri",
...
```

Übungsfrei
```
{
  "title": "Übungsfrei Multi-Unit 1",
  "text": "Dilligaf",
  "group": [
    "Obsidian",
    "Turmalin",
    "Rikki-Tikki",
    "Bratwurscht"
  ],
  "start": "2018-02-07T22:00:00.000Z",
  "end": "2018-02-07T22:00:00.000Z",
  "bringAlong": "free",
  "uniform": "free",
...
```

As can be seen, the structure of the group array differs between those two payloads.

**Comment by peschmae** [comment]
I've adjusted the admin.component.ts, to get rid of the wrapping array for normal tageler, in my own hotfix branch.

But since I now would need to adjust all templates, and want to avoid merge conflicts with #68 (because I'm lazy), I will continue work on this, as soon as the PR is merged

**Comment by peschmae** [comment]
Fixed with #69
